src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -294,6 +302,16 @@ export class HuiDialogEditCard
     this._guiModeAvailable = ev.detail.guiModeAvailable;
   }
 
+  private _handleSetClipboard(
+    ev: HASSDomEvent<SetDashboardClipboardEvent>
+  ): void {
+    ev.stopPropagation();
+    if (this._params?.clipboard) {
+      this._params.clipboard.card = ev.detail.config;
```

Comment:
Admittedly this is kind of funky design here, I went through several iterations of trying to make this work but this is the first thing I ended up on that worked properly. 

I had a very hard time finding any way for clipboard updates in the dialog to be reflected back to the main storage in hui-view. I typically had sent clipboard updates by firing events, but I could not figure out any way to send an event from the dialog back to the hui-view, as they seem to be in different html hierarchies. Tried a bunch of different ways with passing function pointers, passing `this` pointers, couldn't get anything to work. 

So ultimately I created the DashboardClipboard object and passed that to the dialog, and then the dialog could write the clipboard to that directly instead of needing to update the hui-view. 

So you'll see in hui-view and dialogs that the clipboard is an object that contains a LovelaceCardConfig, while in all the editor hierarchies under the dialog the clipboard is just passing a LovelaceCardConfig directly.